/*
 * Copyright (c) 2024, Dylam De La Torre <dyxel04@gmail.com>
 *
 * SPDX-License-Identifier: AGPL-3.0-or-later
 */
#ifndef YGOPEN_PROTO_ROOM_HPP
#define YGOPEN_PROTO_ROOM_HPP
// clang-format off
#include <room.pb.h>
// clang-format on

namespace YGOpen::Proto::Room
{

constexpr size_t MAX_DUELISTS_PER_TEAM = 3;

} // namespace YGOpen::Proto::Room

#endif // YGOPEN_PROTO_ROOM_HPP
